docs/architecture-decisions/adr003-e2e.md

```diff
@@ -0,0 +1,31 @@
+# ADR003: E2E Test Strategy
+
+## Status
+
+accepted
+
+## Context
+
+There is the need to run E2Es automatically in the pipeline to test our whole
+application. We had no strategy on what and where to do any end-to-end testing.
+
+## Decision
+
+- We are only end-to-end testing our SPA application because:
+- 1. it reduces the complexity of the whole project,
+- 2. there is no value in having API E2Es while the SPA E2E cover all
+     functionality,
+- 3. we ship only one bundle and therefore do not need regression tests for
+     multiple API versions.
```

Comment:
```suggestion
- We are only end-to-end testing our SPA application because:
  1. it reduces the complexity of the whole project,
  2. there is no value in having API E2Es while the SPA E2E cover all
     functionality,
  3. we ship only one bundle and therefore do not need regression tests for
     multiple API versions.
```